src/app/authorization/registration/registration.component.css

```diff
@@ -1,33 +1,34 @@
-*{
-    font-family: Roboto;
+* {
+  font-family: Roboto;
 }
-.card{
-    margin: 0 auto; 
-    /* float: none; Added */
-    margin-bottom: 0px;  
-    border-radius: 15px;
-    height: auto;    
-}
-input[type="date"]{
-    margin-left:20px;
-    padding:3px;
-    font-family: "Roboto Mono",monospace;
-    border-radius: 4px;
-    width:320px;
-    margin-top: 10px;
+
+.fs-3 {
+  font-weight: bold;
+  color: rgb(63, 63, 236);
 }
-.cpwd{
-    margin-top: 10px;
+.card {
+  padding: 10px;
+  margin: 0 auto;
+  margin-bottom: 0px;
+  height: auto;
 }
-.gender{
-    margin-top: 10px;
+
+.form-control {
```

Comment:
boostrap class remove thsi

Reply:
I have uses to input filed class name form-control but I have change class name form